finance/ledger/Portfile

```diff
@@ -42,9 +48,10 @@ configure.args-append \
                     -DGMP_PATH=${prefix}/include \
                     -DMPFR_LIB=${prefix}/lib/libmpfr.dylib \
                     -DMPFR_PATH=${prefix}/include \
-                    -DPYTHON_EXECUTABLE=${prefix}/bin/python2.7 \
-                    -DUSE_PYTHON27_COMPONENT=ON \
+                    -DPYTHON_EXECUTABLE=${prefix}/bin/python3.7 \
```

Comment:
You've added a dependency on python38 but you've specified python3.7 on this line. You probably meant to specify python3.8 here.